.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
JSON doesn't take comments. Should probably pull these or add to Readme.

Reply:
While true, that file was generated by and used exclusively by VSCode, so I think it's ok, but I'll remove